eth/api_debug_test.go

```diff
@@ -70,7 +70,7 @@ func TestAccountRange(t *testing.T) {
 	var (
 		statedb  = state.NewDatabaseWithConfig(rawdb.NewMemoryDatabase(), &trie.Config{Preimages: true})
 		state, _ = state.New(types.EmptyRootHash, statedb, nil)
-		addrs    = [AccountRangeMaxResults * 2]common.Address{}
+		addrs    = [maxAccountRangeResults * 2]common.Address{}
```

Comment:
WHy is this changed? If you make a PR to move content from one file into several, then please do not _also_ change stuff. Do purely the move, and if you find any follow-up change, add it in a second PR. Otherwise the review becomes semi-impossible.